frontend/src/bundles/studio/components/video-menu/components/menu-content/script-content/script-content.tsx

```diff
@@ -18,7 +21,14 @@ import { Script, VoicesModal } from './components/components.js';
 
 const ScriptContent: React.FC = () => {
     const dispatch = useAppDispatch();
-    const scripts = useAppSelector(({ studio }) => studio.scripts);
+    const studio = useAppSelector(({ studio }) => studio);
+
+    useEffect(() => {
+        if (studio.voices.length === 0) {
```

Comment:
```suggestion
        if (!voices.length) {
```

Reply:
Yes, I also tried to do that, but ESLint doesn't like that way